config-model/src/main/java/com/yahoo/searchdefinition/derived/ImportedFields.java

```diff
@@ -39,8 +40,17 @@ public void getConfig(ImportedFieldsConfig.Builder builder) {
     }
 
     private static void considerField(ImportedFieldsConfig.Builder builder, ImportedField field) {
-        if (field.targetField().doesAttributing()) {
-            builder.attribute.add(createAttributeBuilder(field));
+        ImmutableSDField targetField = field.targetField();
+        String targetFieldName = targetField.getName();
+        if (targetFieldName.indexOf('.') == -1) {
```

Comment:
Consider adding a helper function (!isNestedFieldName(...)) to better describe what we try to do here.